coordinator/tributary/src/tendermint/tx.rs

```diff
@@ -83,18 +83,15 @@ fn decode_and_verify_signed_message<N: Network>(
 }
 
 // TODO: Move this into tendermint-machine
-// TODO: Strongly type Evidence, instead of having two messages and no idea what's supposedly
-// wrong with them. Doing so will massively simplify the auditability of this (as this
-// re-implements an entire foreign library's checks for malicious behavior).
+// This function takes a TendermintTx, which can't be imported to tendermint create.
```

Comment:
1) This comment is contradictory, assuming it's part of the PR.
2) The underlying evidence checking can be moved.

Reply:
oh yes the evidence checking can be moved I thought TODO you meant the whole function as is. 👍